src/nanoarrow/buffer_test.cc

```diff
@@ -141,9 +141,6 @@ TEST(BufferTest, BufferTestFill) {
   }
 
   ArrowBufferReset(&buffer);
-
```

Comment:
The BufferTestError checks for ENOMEM using `ArrowBufferResize` - do you think it would be better to add the `ArrowBufferReserve` test there for consistency or do you want to keep within this test?